plugin/hardtime.vim

```diff
@@ -18,6 +18,7 @@ call s:check_defined("g:list_of_visual_keys", ["h", "j", "k", "l", "-", "+", "<U
 call s:check_defined("g:list_of_normal_keys", ["h", "j", "k", "l", "-", "+", "<UP>", "<DOWN>", "<LEFT>", "<RIGHT>"])
 call s:check_defined("g:list_of_insert_keys", ["<UP>", "<DOWN>", "<LEFT>", "<RIGHT>"])
 call s:check_defined("g:list_of_disabled_keys", [])
+call s:check_defined("g:list_of_resetting_keys", ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'])
```

Comment:
I don't consider the `5j` as the mistake. The first `j` is the mistake (not doing `6j` in the first place).
The `5j` afterwards is just applying what's actually proposed by this plugin.

But I am fine with making it optional.